app/controllers/reviews_controller.rb

```diff
@@ -41,6 +38,22 @@ def create
     if @review.save
       flash[:notice] = "Review added"
       redirect_to bar_path(@bar)
+      mg_client = Mailgun::Client.new ENV["MAILGUN_API_KEY"]
+      message_params = { from: "system@bevboston.com",
+                         to: "#{@bar.user.email}",
+                         subject: "#{@review.user.email} has added a comment to your #{@bar.name} page on bevboston.",
+                         text: "#{@review.user.email} had this to say about #{@bar.name} '#{@review.title}: #{@review.body}'  You can review their comments here: #{bar_path(@bar)}"
+      }
+      mg_client.send_message "sandbox36abba9a96ae44448899ac794b4f3f33.mailgun.org", message_params
+
+      twitter_client = Twitter::REST::Client.new do |config|
```

Comment:
You need a test for this. Definitely abstract this into a private method - possibly even a separate `TwitterClient` object. I would suggest removing this twitter feature for now, and doing it as part of a different PR that is well tested.

Doing this integration properly with testing is a sizable chunk of work, and you don't want to hold back your mailgun feature because of this.